app/assets/stylesheets/navigation.css.scss

```diff
@@ -25,6 +28,29 @@
   .cta-nav:hover {
     background: darken($cmPurple, 10%);
   }
+
+  @media (max-width: 835px) {
+    .top-bar-section {
+      overflow: hidden;
+      height: 0px;
+    }
+    &.expanded .top-bar-section {
+      overflow: visible;
+      height: auto;
+    }
+    ul.left {
+      margin-top: 0px;
+    }
+  }
+
+  @media (min-width: 768px) and (max-width: 835px) {
+    &.logged-out .toggle-topbar.menu-icon {
+      margin-top: -46px;
+    }
+    ul.left > li.cta .button {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4